src/libs/localStore/userPersistedStorage.ts

```diff
@@ -0,0 +1,39 @@
+import CozyClient from 'cozy-client'
+
+import { storeData, getData } from './storage'
+
+export enum UserPersistedStorageKeys {
+  BackupConfig = 'AmiralAppBackupConfig'
+}
+
+interface UserPersistedStorageItems {
+  backupConfig: object
+}
+
+const formatKey = (
+  client: CozyClient,
+  name: UserPersistedStorageKeys
+): string => {
+  return `@user_${client.getStackClient().uri}_${name}`
```

Comment:
I would prefer to use the "normalised" fqdn here. It is already used in some other places (i.e. bundle config) and it is easier to read (we skip the protocol and remove special chars)

https://github.com/cozy/cozy-flagship-app/blob/718f7be87fc5508e15b473819e6ce6912fa9f559/src/libs/client.js#L123-L124

Reply:
Much better thank you